### PR TITLE
Lpscrypt/work/figma0.5 topbar feedback

### PR DESCRIPTION
**Pull request by LPSCRYPT**
topbar edits

ran npm update which updates many of the package versions

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -86,7 +96,9 @@ function App() {
                   />
                 </h1> */}
                 {/* TODO: move to styling file */}
-                <DomainsGlobal domain={'ROOT'} />
+                <div style={{ position: 'absolute', top: '0' }}>
```
Comment (benefacto) [inline src/App.tsx:99]: If we're using most of our styling in SCSS classes, can we be consistent about that and not use inline styles?

**Review by benefacto — CHANGES_REQUESTED** [review]
Can you merge develop into your branch and resolve any conflicts? Looks like you're two commits behind and we want branches to be up-to-date with develop before we merge them into it. Once that's done and my other comment is resolved, this should be good for me to approve and squash merge in.

File: src/App.tsx (hunk 1 above)
Comment (LPSCRYPT) [reply src/App.tsx:99]: ok

Comment (benefacto) [reply src/App.tsx:99]: Will be addressed in another pull request

**Review by benefacto — APPROVED** [review]
LGTM!

**Pull request merged**